src/Servers/Kestrel/test/InMemory.FunctionalTests/KestrelMetricsTests.cs

```diff
@@ -159,6 +159,7 @@ await connection.ReceiveEnd(
     }
 
     [Fact]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/57944")]
```

Comment:
Do we think quarantining is even necessary this time? I'm confident in the fix.

Reply:
Half the work in the process is creating an issue. You've already done that. May as well follow the rest of the process.